src/Sarif/SdkResources.resx

```diff
@@ -284,4 +284,10 @@ You can also refer to properties in the result's property bag with 'properties.&
   <data name="WRN997_OneOrMoreFilesSkippedDueToSize" xml:space="preserve">
     <value>One or more files were skipped for analysis due to exceeding size limits (currently configured as {0} KB). The 'max-file-size-in-kb' command-line argument can be used to increase this threshold.</value>
   </data>
+  <data name="ERR998_IncompatibleRuleDetected" xml:space="preserve">
+    <value>Detected incompatible rules. Rule '{0}' is incompatible with rule '{1}'.</value>
+  </data>
+  <data name="Wrn998_IncompatibleRuleDetected" xml:space="preserve">
```

Comment:
>Wrn998 [](http://example.com/codeflow?start=14&length=6)

just question, 
how do we choose 998 or 997, I see a few 997 above.

Reply:
"Wrn998_IncompatibleRuleDetected" should capitalize the first 3 letters to be more consistent with other name.

Reply:
It is in Warnings.cs line 21, the WRN998 is the category for "// (Non-catastrophic) conditions that result in rules disabling themselves."